tests/integration/helpers.py

```diff
@@ -144,26 +135,36 @@ def access_dashboard(
 
 def access_dashboard_https(host: str, password: str):
     """This function should be rather replaced by a 'requests' call, if we can figure out the source of discrepancy."""
-    curl_cmd = check_output(
-        [
-            "bash",
-            "-c",
-            'curl  -XPOST -H "Content-Type: application/json" -H "osd-xsrf: true" -H "Accept: application/json" '
-            + f"https://{host}:5601/auth/login -d "
-            + "'"
-            + '{"username":"kibanaserver","password": "'
-            + f"{password}"
-            + '"'
-            + "}' --cacert ca.pem",
-        ],
-        text=True,
-    )
+    try:
+        curl_cmd = check_output(
+            [
+                "bash",
+                "-c",
+                'curl  -XPOST -H "Content-Type: application/json" -H "osd-xsrf: true" -H "Accept: application/json" '
+                + f"https://{host}:5601/auth/login -d "
+                + "'"
+                + '{"username":"kibanaserver","password": "'
+                + f"{password}"
+                + '"'
+                + "}' --cacert ca.pem --verbose",
+            ],
+            stderr=subprocess.STDOUT,
+            text=True,
+        )
+    except subprocess.CalledProcessError as err:
+        logger.error(f"{err}, {err.output}")
+        return False
     return "roles" in curl_cmd
 
 
-async def access_all_dashboards(ops_test: OpsTest, relation_id: int, https: bool = False):
+async def access_all_dashboards(
```

Comment:
```suggestion
async def access_all_dashboards_units(
```

Reply:
May I keep internal helper functions names as they are -- except when particularly strong reason for otherwise?

Please note that we're modifying an 1.5 h pipeline where the smallest typo may take 45mins to notice.
(And that we were hoping to merge this code last week already...)